app/src/components/market/common/market_scale/index.tsx

```diff
@@ -489,7 +489,7 @@ export const MarketScale: React.FC<Props> = (props: Props) => {
       subtitle: startingPointTitle,
       xValue: currentPrediction
         ? Number(currentPrediction)
-        : calcXValue(startingPoint || new BigNumber(0), lowerBound, upperBound, 18),
+        : calcXValue(startingPoint || new BigNumber(0), lowerBound, upperBound, 18) / 100,
```

Comment:
Does this value 18 represent token decimals? If so...dangerous to hardcode to 18

Reply:
Thanks for pointing that out. The decimal value is always 18 since it just represents values unrelated to tokens. I've removed the decimals parameter to reflect this